.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Motivation
+<!-- How does this code change improve the world? Could be a link to JIRA -->
```

Comment:
"Could be a link to JIRA" will be confusing to third parties, briefly explain that this is a hint for IA people?